examples/README-authz.md

```diff
@@ -0,0 +1,403 @@
+## Token Based Authorization with Keycloak Authorization Services
+
+Once the Kafka Broker has obtained an access token by using Strimzi Kafka OAuth for authentication, it is possible to use centrally managed authorization rules to enforce access restrictions onto Kafka Clients.
+For this Strimzi Kafka OAuth supports the use of Keycloak Authorization Services.
+
+A custom authorizer has to be configured on the Kafka Broker to take advantage of Authorization Services REST endpoints available on Keycloak, which provide a list of granted permissions on resources for authenticated users.
+The list is fetched once, and enforced locally on the Kafka Broker for each user session in order to provide fast authorization decisions.
```

Comment:
> The list is fetched once

Is it once per user session? I.e. disconnecting it and connecting again would get a fresh list? Would some time based limit make more sense here? This could be a security issue for many users.

Reply:
Yes, it is once per connection (Kafka broker session) currently.

I guess adding periodic refresh is the next thing to add as @tombentley has also [expressed concern about this](https://github.com/strimzi/strimzi-kafka-oauth/pull/24#discussion_r367803753).